commands/admin/settings.js

```diff
@@ -24,96 +27,154 @@ module.exports.load = (client) => {
           msg.setFooter(`${message.author.username}#${message.author.discriminator}`, message.author.avatarURL)
           msg.setTimestamp()
           message.channel.send(msg)
-        } else {
-          if (args[0] === 'welcomes' && args[1] === 'on' && args.length === 2) return toggleSwitch(true, 'welcome_toggle')
-          if (args[0] === 'welcomes' && args[1] === 'off' && args.length === 2) return toggleSwitch(false, 'welcome_toggle')
-          if (args[0] === 'welcomes' && args.length !== 2) return client.errorMessage(message, `Welcomes can be either toggled on or off. Example: ${client.settings.prefix}settings welcomes on`)
-
-          if (args[0] === 'practice_channels' && args.length === 1) return client.errorMessage(message, `Practice channels can be add by using \`${client.settings.prefix}settings practice_channels add {#channel}\` and removed with \`del {#channel}\``)
-          if (args[0] === 'practice_channels' && args.length === 3 && args[1] === 'add' && isValidChannel(args[2])) return addChannelPractice(args[2])
-          if (args[0] === 'practice_channels' && args.length === 3 && args[1] === 'del' && isValidChannel(args[2])) return delChannelPractice(args[2])
+          return
+        }
 
-          if (args[0] === 'dm_welcomes' && args[1] === 'on' && args.length === 2) return toggleSwitch(true, 'dm_welcome_toggle')
-          if (args[0] === 'dm_welcomes' && args[1] === 'off' && args.length === 2) return toggleSwitch(false, 'dm_welcome_toggle')
-          if (args[0] === 'dm_welcomes' && args.length !== 2) return client.errorMessage(message, `dm_welcomes can be either toggled on or off. Example: ${client.settings.prefix}settings dm_welcomes on`)
+        switch (args[0]) {
+          case 'welcomes':
+            return interpretOnOffToggle('welcome_toggle', args, `Welcomes can be either toggled on or off. Example: ${client.settings.prefix}settings welcomes on`)
+
+          case 'practice_channels':
+            if (args.length != 3) {
+              return client.errorMessage(message, `Practice channels can be added by using \`${client.settings.prefix}settings practice_channels add <#channel>\` and removed with \`del <#channel>\``)
+            }
+
+            switch (args[1]) {
+              case 'add':
+                if (isValidChannel(args[2])) {
+                  return addChannelPractice(args[2])
+                }
+                break
+              case 'del':
+                if (isValidChannel(args[2])) {
+                  return delChannelPractice(args[2])
+                }
+                break
+              default:
+                return client.errorMessage(message, `Practice channels can be add by using \`${client.settings.prefix}settings practice_channels add <#channel>\` and removed with \`del <#channel>\``)
+            }

```

Comment:
Missing break statement here. If we were adding/deleting but the channel is not valid, there is no error, which will fail to return and fall through to `dm_welcomes`.